projects/igniteui-angular/src/lib/grids/grid/grid.component.spec.ts

```diff
@@ -2144,8 +2144,7 @@ describe('IgxGrid Component Tests', () => {
                 expect(cell.inEditMode).toBe(false);
                 expect(cell.value).toBe(110); // SORT does not submit
 
-                expect(gridAPI.escape_editMode).toHaveBeenCalled();
```

Comment:
expect(gridAPI.escape_editMode).toHaveBeenCalled(); should exist in the test.

Reply:
It exists on next line doesnt it